src/components/Button/FolderButton/FolderButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { MouseEvent, ReactNode } from "react";
+import styles from "./folderButton.module.css";
+
+interface Props {
+  children: ReactNode;
+  onClick: (e: MouseEvent<HTMLButtonElement>) => void;
+  ischecked: boolean;
+}
+
+function FolderButton({ children, onClick, ischecked }: Props) {
+  return (
+    <button
+      className={`${styles.btn} ${ischecked ? styles.checked : ""}`}
```

Comment:
className을 여러개 적용하는 경우와, 조건식을 사용할 때는 이렇게 해주기보다 `classnames` 혹은 `clsx`와 같은 라이브러리를 사용하면 편리하게 적용할 수 있어요~
```suggestion
      className={classnames(styles.btn, { [styles.checked] : isChecked })}
```

그리고 classnames에서 제공하는 bind 메소드를 활용해주면 `styles.` 처럼 앞에 붙여주는 구문도 제거할 수 있어요~ [참고자료](https://uncertainty.oopy.io/43ef0ac4-cddc-460d-90d8-6a418f5f42ab)